test/fluree/db/ledger/docs/identity/signatures_test.clj

```diff
@@ -0,0 +1,204 @@
+(ns fluree.db.ledger.docs.identity.signatures-test
+  (:require [clojure.test :refer :all]
+            [fluree.db.test-helpers :as test]
+            [fluree.db.ledger.docs.getting-started.basic-schema :as basic]
+            [fluree.db.api :as fdb]
+            [org.httpkit.client :as http]
+            [clojure.core.async :as async :refer [<!!]]
+            [fluree.db.util.json :as json]
+            [fluree.db.query.http-signatures :as http-signatures]))
+
+;; Use fdb-api-open = false, everything that goes through the endpoints needs to be signed
+(use-fixtures :once
+              (partial test/test-system
+                       {:fdb-api-open          false
+                        :fdb-group-servers     "ABC@localhost:11002"
+                        :fdb-group-this-server "ABC"
+                        :fdb-storage-type      "memory"
+                        :fdb-api-port          @test/port}))
+
+
+(def endpoint (str "http://localhost:" @test/port "/fdb/"))
+
+
+(defn send-parse-request
+  [ledger type req opts]
+  (-> req
+      (test/standard-request opts)
+      (->> (http/post (str endpoint ledger "/" type)))
+      deref
+      :body
+      json/parse))
+
+
+;; Create auth records -> root, all persons, all persons no handles
+(defn create-auths [ledger]
```

Comment:
teeny tiny style thing. (i think) we decided to put the argument vector on a new line after the `defn` keyword and fn name.